Assets/Scripts/Managers/AudioManager.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using GameAudio;
+using UnityEngine;
+
+namespace Managers
+{
+    /**
+     * Play sounds with 'AudioManager.Instance.Play("name");'
+     */
+    public class AudioManager : Singleton<AudioManager>
+    {
+        [SerializeField][Tooltip("Container for all sounds")]
+        private List<Sound> sounds;
+
+        [SerializeField]
+        private GameObject audioSources;
+
+        private void Start()
```

Comment:
```suggestion
        protected override void Awake()
        {
            base.Awake():
        }
```
You can override the awake function of the singleton. Make sure to call `base.Awake()`

Reply:
thanks, will try!